ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -246,6 +246,78 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws-disruptive,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-aws-fips
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
```

Comment:
@sallyom once you use the CI Operator configuration to generate this job, remove the entry in the configuration and remove this label from the job, which will allow the generator to ignore it in the future and let you make the `ENABLE_FIPS` change as you want.

Reply:
ok